queries/orderbook/barn_batch_rewards.sql

```diff
@@ -273,16 +284,37 @@ primary_rewards as (
     GROUP BY
         solver
 ),
+integrator_fees_per_solver AS (
+    SELECT
+        solver,
+        protocol_fee_recipient,
+        sum(protocol_fee * protocol_fee_token_native_price) as protocol_fee_in_eth
```

Comment:
To be consistent in the naming:
```suggestion
        sum(protocol_fee * protocol_fee_token_native_price) as protocol_fee_eth
```

Reply:
Done